Scripts/NuGet/EdFi.Installer.WebApi/test.ps1

```diff
@@ -164,6 +179,7 @@ function Invoke-UninstallDifferentWebSite {
 
 try {
     switch ($Scenario) {
+        "DifferentPackageSource" { Invoke-DifferentPackageSource } 
```

Comment:
need to update the default case to include this new scenario

Reply:
I updated it  .Thanks